File: Backend/App/models/visit.py
from sqlalchemy import Column,String,Integer,Boolean,ForeignKey,DateTime
from Backend.App.core.database import Base
from Backend.App.models.person import Person

class Visit(Base):
    __tablename__ ="Visit"

    VisitId = Column(Integer,primary_key=True,index=True)
    PatientId=Column(Integer,ForeignKey(Person.PersonId))
    Date_Of_Visit = Column(DateTime)

File: DotNetBackend/ViveksHomoeoClinic/ViveksHomoeoClinic/Models/EFModel/Visit.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ViveksHomoeoClinic.Models.EFModel
{
    public class Visit
    {
        [Key]
        public int VisitId { get; set; }
        [Required]
        public int PatientId { get; set; }

        [ForeignKey(nameof(PatientId))]
        public Patient Patient { get; set; }
        public DateTime DateOfVisit { get; set; }
    }
}
